package/SUSEConnect.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Tue Nov 16 14:52:50 UTC 2021 - Thomas Muntaner <tmuntaner@suse.com>
+
+- Update to 0.3.33
+- Install systemd dependency on aarch64 systems.
```

Comment:
Hmm wondering weather we want to pull in all of systemd in this case, or instead, handle the exception gracefully, and report the system as physical by default. 
I suppose there is a reason systemd is not present on those systems, and I guess they won't always want to install it (e.g. you don't want systemd in a container).

Reply:
@lagartoflojo I have no issues if the PR is changed and another dependency is placed here instead. I just want to prevent the case I saw yesterday of being brought into a call of multiple engineers trying to figure out why SUSEConnect doesn't work with their new apple computers.

If you look at the git history, we didn't require systemd before, but I guess business requirements changed. Maybe whoever takes over this card can also look into whether or not we should go back to an old method.

Reply:
I'd support setting the hypervisor to `nil` in case there's no `systemd-detect-virt` installed, because installing systemd on all systems can have consequences we don't oversee.